tests/integration/integration-test.sh

```diff
@@ -168,8 +175,27 @@ load_test() { #
   rm output.log
 }
 
+### CREATE IMAGEPULLSECRET ####################################
+create_imagepullsecret() { # NAMESPACE # CREATE
+  local CREATE=${2:-true}
+  if $CREATE; then
+    echo -n "Creating Namespace '${1}'..."
```

Comment:
nit: consider using the `${SUCCESS}` and `${FAILED}` shortcuts. or do it your way 🤷 same for the other `echo` outputs.